packedForest/test/uniformStructureRerFTest.R

```diff
@@ -0,0 +1,20 @@
+
+
+h <- read.table('heights.txt')[[1]]
```

Comment:
not sure these txt file are even in the repo...?

Reply:
See https://github.com/neurodata/RerF/blob/jlp/s-rerf_python/packedForest/test/fpTests/binnedForest/processingNodeBinTest.h#L236

Reply:
We hadn't quite figured out how to run the appropriate test.  There is code that generates those files that is commented out, and the R-script is there to run manually.